lib/tree.rb

```diff
@@ -1,38 +1,55 @@
 class NoApplesError < StandardError; end
 
 class AppleTree
-  attr_#fill_in :height, :age, :apples, :alive
+  attr_accessor :height, :age, :apples, :alive
 
   def initialize
+    @height = 0
+    @age = 0
+    @apples = 0
```

Comment:
This makes it sound like `@apples` will be an array of apples.  I'd say either switch the value or change the name to make clear that it's a count.

Reply:
Rupert ended up doing the array design, it works out much better overall